### PR TITLE
Added Windows version for step 3 under Development

### PR DESCRIPTION
**Pull request by AdhirRamjiawan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ If you don't care how and why and just want something that works, follow these s
 
 1. [fork this repo][fork]
 1. `go get github.com/exercism/cli/exercism`
-1. `cd $GOPATH/src/github.com/exercism/cli`
+1. `cd $GOPATH/src/github.com/exercism/cli` (or `cd %GOPATH%/src/github.com/exercism/cli` on windows)
```
Comment (nywilken) [inline CONTRIBUTING.md]: `windows` should be capitalized.

**Comment by AdhirRamjiawan** [comment]
Thanks @nywilken, I've updated it to "Windows".

**Comment by nywilken** [comment]
@AdhirRamjiawan thanks for updating the documentation. The change looks good.

Before I merge, could you please update your PR to follow the Exercism guidelines for [commit messages](https://github.com/exercism/docs/blob/master/contributing/git-basics.md#commit-messages). 
Feel free to give me a holler if you have questions or run into any issues. Thanks!

**Comment by robphoenix** [comment]
Thanks for this @AdhirRamjiawan 
Following on from @nywilken's request, could you rebase and squash these commits into a single one, please. Thanks. And please do say if there's any issues with this. Goodness knows I regularly mess my commits up! :laughing:

**Comment by AdhirRamjiawan** [comment]
Thanks @robphoenix and @nywilken. Commits squashed and PR and commit message updated.

**Comment by nywilken** [comment]
@AdhirRamjiawan thanks again for the contribution and your patience. Your changes have been merged.

**Comment by AdhirRamjiawan** [comment]
thanks @nywilken . It's really a pleasure! I'd love to continue contributing to this project. If there's anything that I can help with please let me know.

**Comment by nywilken** [comment]
@AdhirRamjiawan That's great news! We'd love to have you contribute.  We currently have a number of open issues labeled "good first patch" on the exercism org: 

‪https://github.com/search?utf8=%E2%9C%93&q=org%3Aexercism+label%3A%22good+first+patch%22+is%3Aopen&type=‬ 

There is a ton of opportunity in various language tracks in preparation for the new 
v2 exercism 
‪https://github.com/exercism/v2-feedback/tree/master/README.md‬ 

If there a specific Exercism project, the cli included, that you would like to contribute to, start by looking at the issues to get a sense of things. If you find an issue you would like to take on, leave a comment and we will provide any help necessary to get you going. If you have a non-issue contribution in mind open a [PR](https://github.com/exercism/docs/blob/master/contributing/pull-request-guidelines.md) for us to review. 

More information on contributing  can be found at https://github.com/exercism/docs/

Thanks again for your time and effort.

**Comment by AdhirRamjiawan** [comment]
@nywilken , awesome, thanks!

**Pull request merged**